CONTRIBUTING.md

```diff
@@ -88,7 +88,7 @@ $> swift --version
 Apple Swift version 5.3
 ```
 
-Note: Alternatively use tools like [swiftenv](https://github.com/kylef/swiftenv) that help manage toolchains versions.
+Note: Alternatively, there are tools like [swiftly](https://github.com/swiftlang/swiftly) that can instll and manage toolchains automatically. This repository has a file called `.swift-version` that will keep swiftly at the current recommended version of the toolchain for best results.
```

Comment:
nit: This line should explain how to use swiftly to pull the correct toolchain.

Also, typo: `instll`